### DISCUSSION TITLE
Getting Started This Process Step has already occurred.

### DISCUSSION THREAD

**Discussion by ahmedattiq** [flowsforapex/apex-flowsforapex#585] [Q&A] [answered]
Hi, 
I have just started working on flow for apex and followed step by step document to the dot _ However, I am getting below error when completing the sample flow 

1 error has occurred
This Process Step has already occurred. (Incorrect step key "null" supplied while expecting step key umjIfvdnNN).

(reference: https://flowsforapex.org/latest/getting-started/)_

![image](https://user-images.githubusercontent.com/108540545/224057660-f5ecf4dd-af58-400b-9ba9-f4bf954f7253.png)

**Comment by LouisMoreaux** [comment, marked as answer]
Hi,

I think there are some issues on the Getting Started page, I am sorry about it and we will fix it.

Based on the ticket description, I think you missed the column sbfl_step_key on the task list (page 1) and then you should update the link from the report to page 3 to populate P3_STEP_KEY with the value of the SBFL_STEP_KEY column.

Hope it will help you to go ahead and keep experimenting!

Louis

**Comment by ahmedattiq** [comment]
Hi @LouisMoreaux thank you for the prompt response. I will try the suggestion. However I dont find any reference of setting up column sbfl_step_key in the Getting started document which i am following 

https://flowsforapex.org/latest/getting-started/

**Reply by LouisMoreaux** [reply]
Yes this is an issue with the Getting Started instructions, I will address that issue. Thank you for opening this discussion

**Comment by ahmedattiq** [comment]
Thank you @LouisMoreaux it worked after adding the sbfl_step_key column in the task list 

![image](https://user-images.githubusercontent.com/108540545/224064613-22e3a4a5-7a76-4c81-9c34-167e7a637da2.png)
